modules/PhyVersoBSP/phyverso_mcu_comms/evSerial.h

```diff
@@ -41,6 +101,7 @@ class evSerial {
     sigslot::signal<int, PpState> signal_pp_state;
     sigslot::signal<FanState> signal_fan_state;
     sigslot::signal<int, LockState> signal_lock_state;
+    sigslot::signal<int, const std::vector<uint16_t>&> signal_psensor_data;
```

Comment:
Maybe maybe call it `system_specific_data` as well?
or "generic_array" as in other cases?